.github/workflows/tests.yml

```diff
@@ -0,0 +1,37 @@
+name: Tests
+
+on: ['push', 'pull_request']
```

Comment:
Maybe only on pull requests, plus workflow dispatch? So that pushes to arbitrary branches (with no associated PR) do not trigger the event?

E.g. https://github.com/multiversx/mx-sdk-js-core/blob/main/.github/workflows/build.yml#L7

Reply:
updated 👍